samples/README.md

```diff
@@ -0,0 +1,70 @@
+# .NET Framework Docker Samples
+
+The samples show various ways to use .NET Framework and Docker together. You can use the samples as the basis of your own Docker images or just to play.
+
+The samples exercise various levels of functionality. The [.NET Framework Docker sample](dotnetapp/README.md) includes the most functionality, including build, unit testing, and pushing images to a container registry. The [ASP.NET Core Docker sample](aspnetapp/README.md) includes instructions for testing images with [Azure Container Instances](https://azure.microsoft.com/services/container-instances/). The samples include detailed instructions for use with and without Docker.
```

Comment:
I see a lot of `Core` refernces throughout the docs.

Reply:
Yes. Those are the docs I started with. I think I got most of the "Core" references.